lib/Raven/ReprSerializer.php

```diff
@@ -29,7 +29,13 @@ protected function serializeValue($value)
         } elseif (is_integer($value) || is_float($value)) {
             return (string) $value;
         } elseif (is_object($value) || gettype($value) == 'object') {
-            return 'Object '.get_class($value);
+            $repr = 'Object ' . get_class($value);
+            if (method_exists($value, 'toSentry')) {
+                $repr .= ' ' . $this->serializeValue($value->toSentry());
+            } elseif (method_exists($value, 'toDebugContext')) {
+                $repr .= ' ' . $this->serializeValue($value->toSentry());
```

Comment:
I think you went a bit to quick here, should be `$value->toDebugContext()` here right?